lib/far_mar/market.rb

```diff
@@ -0,0 +1,15 @@
+CSV.read("./support/markets.csv")
```

Comment:
I know this is just your preliminary code, but avoid reading the same file multiple times in your later implementation :-)